content/TrueCommand/SystemManagement/_index.md

```diff
@@ -13,35 +13,34 @@ For information on the Top Bar and its options, refer to the [Interface Overview
 ### System Cards
 
 Each system has a unique card to display statistics.
-When the system has an alert, an *Alerts* bubble appears next to the system version to show how many alerts there are for that specific system.
+When the system has an alert, an **Alerts** bubble appears next to the system version to show how many alerts there are for that specific system.
 See [Alert Management]({{< relref "/TrueCommand/Alerts/AlertManage.md" >}}) for further information.
 
 ![DashboardSingleSystemView](/images/TrueCommand/2.0/DashboardSingleSystemView.png "Dashboard Single System View")
 
+The **Storage** graph shows how many pools and drives the system is using. It also displays used and available storage by size and percentage.
 
-*Storage* lists how many pools and drives are being used by the system. It also shows how much storage is used and available by size and percentage.
+**ARC MISS** shows how often the system is using disks instead of the ARC cache. Anything above 0% means that the system is using RAM. The numbers vary by use case and workload.
 
-*ARC MISS* shows how often the system is using disks instead of the ARC cache. Anything above 0% means that the system is using RAM. The numbers vary on use case and work load.
-
-**There are also several "hot spots" on the card that will open system specific areas for management.** 
+**There are also several "hot spots" on the card that open system-specific areas for management.** 
```

Comment:
suggested change based on best practices, which is to never use double quotes " " for emphasis, change to italics is the recommendation.